docs/csharp/tour-of-csharp/index.md

```diff
@@ -80,7 +80,7 @@ The following outline provides an overview of C#'s type system.
   - [Delegate types](../language-reference/builtin-types/reference-types.md#the-delegate-type)
     - User-defined types of the form `delegate int D(...)`
 
-C# programs use *type declarations* to create new types. A type declaration specifies the name and the members of the new type. Six of C#'s categories of types are user-definable: class types, struct types, interface types, enum types, delegate types, and tuple value types.
+C# programs use *type declarations* to create new types. A type declaration specifies the name and the members of the new type. Six of C#'s categories of types are user-definable: class types, struct types, interface types, enum types, delegate types, and tuple value types. You can also declare `record` types, either `record struct`, or `record class`. Record types have additional synthesized members. You should use records primarily for storing values, with minimal associated behavior.
```

Comment:
> either `record struct`, or `record class`.

For `record class` the `class` is optional. Should this be pointed out?
Or do we want to add the `class` to the `record` in order to make it more explicit and distinct from `record struct`?

Reply:
Also a great conversation. I'm going to leave this as is, with `record struct` and `record class`, but not resolve the conversation because I want other reviewers to provide their thoughts as well.

The features for `record struct` and `record class` are similar, and a single description works for both of them. I've used the word "record" to talk about features that apply to both. I've used `record class` and `record struct` where I mean one or the other. In the opening of the article, I did explain that `class` is optional: `record` and `record class` are the same.